lib/db/DB.js

```diff
@@ -16,19 +19,61 @@ class DB {
   }
 
   define() {
-    this.Order = this.sequelize.define('order', {
-      id: { type: Sequelize.INTEGER, primaryKey: true, autoIncrement: true },
-      peerId: Sequelize.INTEGER,
-      invoice: Sequelize.STRING,
-      quantity: Sequelize.DECIMAL(14, 8),
-      price: Sequelize.DECIMAL(9, 8),
+    // P2P
+
+    this.Peer = this.sequelize.define('peers', {
```

Comment:
We can simply define the model as `peer` as it will automatically get pluralized.

Reply:
I prefer to see in the code what I want to get and not be depended on magic tricks, which are most confusing.

Reply:
Ok, either way works so I will defer to you, I was mainly basing off the approach in the sequelize api docs & examples.

Reply:
I went with your approach. Better to be compatible with the documentation ;)